nxapi/nxlog.py

```diff
@@ -77,3 +126,75 @@ def explain_nxlog(nxlog):
 
     return explain
 
+def unify_date(date):
```

Comment:
You might want to use [dateutil](https://dateutil.readthedocs.io/en/stable/parser.html) instead of home-brewing something ;)